src/ape_run/_cli.py

```diff
@@ -213,7 +213,7 @@ def get_command(self, ctx, name):
 
         # NOTE: don't return anything so Click displays proper error
 
-    def result_callback(self, result, interactive):
+    def result_callback(self, result, interactive: bool):  # type: ignore[override]
```

Comment:
we just be hacking click a bit here, dont mind us, mypy

Reply:
the hack is annotating `interactive`?

Reply:
the hack is having `interactive` at all, it isnt in the original method. There are no real changes here.